adam_dunlap.cfdg

```diff
@@ -0,0 +1,34 @@
+startshape clock
+
```

Comment:
I do really like the design.  Did you ever try making the canvas bigger and seeing how far down you could see it?